cuda_core/examples/strided_memory_view.py

```diff
@@ -91,6 +91,7 @@
     gpu_prog = Program(gpu_code, code_type="c++")
     # To know the GPU's compute capability, we need to identify which GPU to use.
     dev = Device(0)
+    dev.set_current()
```

Comment:
Turns out this is a bug fix for CUDA 11, only discovered now. On CUDA 11 when we load the kernel it requires a valid CUDA context (contextless loading is a CUDA 12 feature).

Reply:
good catch

Reply:
It's caught by the CI not by me. This code path is now executed after CuPy is installed in the test stage, and without this fix the CI would not pass.